test/unit/project_test.rb

```diff
@@ -946,5 +946,24 @@ def test_update_first_letter
     project.topic_annotations = 'Chemistry'
     assert project.controlled_vocab_annotations?
   end
-  
+
+  test 'total asset size includes blobs and git repos without duplication' do
+    project1 = FactoryBot.create(:project)
+    project2 = FactoryBot.create(:project)
+    df = FactoryBot.create(:data_file, projects: [project1, project2])
+    workflow1 = FactoryBot.create(:local_git_workflow, projects: [project1, project2])
+    disable_authorization_checks { workflow1.save_as_new_git_version }
```

Comment:
I think it would be good to do this as an additional step under the current assertions, just to confirm the `total_asset_size` is the same before and after adding a new version.

Reply:
See [1c87590](https://github.com/seek4science/seek/pull/1630/commits/1c8759079cf31dac98459dc03fce4fd96e8f0485).